local_vars/local_vars.cc

```diff
@@ -70,7 +71,22 @@ class LocalNameInserter {
             }
             unique_ptr<ast::Reference> refExpImpl(refExp);
             arg.release();
-            namedArgs.emplace_back(nameArg(move(refExpImpl)));
+            auto named = nameArg(move(refExpImpl));
+            for (auto &prev : namedArgs) {
+                if (prev.name == named.name && !absl::StartsWith(prev.name.data(ctx)->shortName(ctx), "_")) {
```

Comment:
The `shortName` function allocates a new string, which isn't great, but this should _only_ be necessary when we have repeated argument names to begin with, which is very rarely—usually, we're just comparing two `NameRef`s, which is much cheaper, and they'll almost never be identical.

Reply:
> The shortName function allocates a new string

it doesn't. It gives a `string_view`